### PR TITLE
New Get backend information page

### LINKED ISSUE #112: [ REFACTOR/NEW PAGE ] Get Backend Information
## What the new page should cover:
combine [list backends, filter backends, find backend info from other channels](https://docs.quantum-computing.ibm.com/run/backends#list-backends), also some info from [gathering_system_information](https://github.com/Qiskit/documentation/blob/4965987dda8319bd368044efde80694ed6252a4f/docs/run/__gathering_system_information.ipynb)

### PR DESCRIPTION
**Pull request by abbycross**
Closes #112.

### CODE REVIEW

**Comment by abbycross** [comment]
Noting that we already have the current [View backend information](https://docs.quantum-computing.ibm.com/run/backend-information) page duplicated within the [System information](https://docs.quantum-computing.ibm.com/run/hardware) page right above it in the toc.... possible next steps:

-  the View backend information page stays contained within the System information page
- the View backend information page gets integrated into the new Get backend information page
- the System information page gets integrated into the Get backend information page

Thoughts? @javabster

**Comment by javabster** [comment]
I think something went wrong when we were doing this issue: https://github.com/Qiskit/documentation/issues/111

I think this is what needs to happen with these pages:

page name: Processor types
should include: only information about the processors, so the system configuration section should be removed

page name: system information
should include: getting information about systems that you don't need code to retrieve. So this would include the "system configuration" section that was previously in processor types, the "system versioning" and "how to cite a system" that are currently in the page

page name: Get backend information using Qiskit (I'm open to alternative naming)
should include: anything about retrieve backend information using code. So this would include the "view system properties" section (that is currently duplicated in 2 places) as well as the information that was originally in scope for this issue (filter backends, gathering system information etc.)

File: docs/run/get-backend-information.ipynb [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+   "outputs": [],
+   "source": [
+    "service.backends(simulator=False, operational=True, min_num_qubits=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A similar method is `least_busy()`, which takes the same filters as `backends()` but returns the backend that matches the filters and has the least number of jobs pending in the queue:"
```
Comment (Eric-Arellano) [inline docs/run/get-backend-information.ipynb:80]:
```suggestion
    "A similar method is [`QiskitRuntimeService.least_busy()`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#least_busy), which takes the same filters as `backends()` but returns the backend that matches the filters and has the least number of jobs pending in the queue:"
```

File: docs/run/hardware.mdx [hunk 2]
```diff
@@ -1,101 +1,100 @@
...
 title: System information
-description: Information about IBM Quantum hardware, features of different processors, and system calibration, properties, and versioning
+description: Information about IBM Quantum system calibration, properties, and versioning
 
 ---
 
 # System information
 
-IBM Quantum offers both open and premium access to a wide variety of quantum systems. All quantum systems deployed by IBM Quantum are based on superconducting qubit technology, as the control and scalability of this technology pave a clear path to achieving quantum advantage with these systems. You can see the full details of all IBM Quantum systems on the [Compute resources page](https://quantum-computing.ibm.com/services/resources?tab=systems).
+IBM Quantum offers both open and premium access to a wide variety of quantum systems. All quantum systems deployed by IBM Quantum are based on superconducting qubit technology, as the control and scalability of this technology pave a clear path to achieving quantum advantage with these systems. You can see the full details of all IBM Quantum systems on the [Compute resources page.](https://quantum-computing.ibm.com/services/resources?tab=systems)
```
Comment (Eric-Arellano) [inline docs/run/hardware.mdx:9]: Why this change? Punctuation should be in the hyperlink?

File: docs/run/hardware.mdx [hunk 3]
```diff
@@ -1,101 +1,100 @@
...
+    *   **Usage** - Click the link to see the jobs that you have run on this system.
 
-Each system has a version number in the form X.Y.Z (major.minor.revision). A circuit compiled for a given system version number is guaranteed to run on that system. If the revision number changes, the circuit will continue to run. If the major or minor number changes, the circuit is not guaranteed to run, although it may do so. The conditions under which a version number may change are listed below:
+## View system configuration
 
-### Major version
+View system configuration values by selecting a system on the [Compute resources page.](https://quantum-computing.ibm.com/services/resources?services=systems) The three tabs in the Calibration data section let you choose how to view the calibration data; the Map view tab is automatically selected.
 
-The major version will increment for system changes such as:
+![../../../\_images/exp-card.png](/images/migration/exp-card.png)
```
Comment (Eric-Arellano) [inline docs/run/hardware.mdx:78]: What's up with the `[]` portion? That is usually supposed to be a human readable description

File: docs/run/hardware.mdx [hunk 4]
```diff
@@ -1,101 +1,100 @@
...
-*   Sample changes.
-*   Major changes to the control electronics.
-*   Moving the system to a new location, if significant behavior changes result.
+An expanded card for a sample system.
 
-### Minor version
+Click the download icon in the upper right of any tab to download a CSV file of calibration data.
 
-The minor version will increment for changes such as:
+![../../../\_images/graph-view1.png](/images/migration/graph-view1.png)
```
Comment (Eric-Arellano) [inline docs/run/hardware.mdx:84]: Ditto file name

File: docs/run/hardware.mdx [hunk 5]
```diff
@@ -1,101 +1,100 @@
...
 
-*   Warmup / cool-down cycles.
-*   Swapping out some electronics, if the replacement appreciably affects operation.
-*   Changing the direction of a controlled-NOT gate.
-*   Dropping a gate for some duration of time due to calibration issues, and corrections cannot readily be done in software.
+The Graph view tab.
 
-### Revision version
+![../../../\_images/table-view.png](/images/migration/table-view.png)
```
Comment (Eric-Arellano) [inline docs/run/hardware.mdx:88]: Ditto file name

**Review by Eric-Arellano — APPROVED** [review]
Nice work!

File: docs/run/hardware.mdx (hunk 2 above)
Comment (abbycross) [reply docs/run/hardware.mdx:9]: When the period comes after the hyperlink, it puts an extra space there, and can also mean a linebreak between the link and the period depending on the line length.

File: docs/run/hardware.mdx (hunk 3 above)
Comment (abbycross) [reply docs/run/hardware.mdx:78]: no idea! This was pre-existing... what should it be?

Comment (Eric-Arellano) [reply docs/run/hardware.mdx:78]: A concise human readable description of the image, like alt text. "ibm_quito system page" for example

Comment (abbycross) [reply docs/run/hardware.mdx:78]: Ah I understand -- I was misreading the code and hadn't realized this is what was rendering in the html 😆 ok let me look at that

File: docs/run/get-backend-information.ipynb [hunk 6]
```diff
@@ -0,0 +1,476 @@
+{
```
[the remark this answers is not in the document]
Comment (abbycross) [reply docs/run/get-backend-information.ipynb:1]: resolved in the URL audit #186 work.

File: docs/run/get-backend-information.ipynb [hunk 7]
```diff
@@ -0,0 +1,476 @@
...
+    "To view the backends you have access to, you can either view a list on the [Compute resources page,](https://quantum-computing.ibm.com/services/resources?tab=yours) or you can use the `backends()` method. This method returns a list of `IBMBackend` instances:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "service.backends()\n",
```
[the remark this answers is not in the document]
Comment (abbycross) [reply docs/run/get-backend-information.ipynb:31]: Added to the first code block!
